Sources/Sentry/SentryBaggage.m

```diff
@@ -44,14 +42,11 @@ - (NSString *)toHTTPHeader
     if (_environment != nil)
         [information setValue:_environment forKey:@"sentry-environment"];
 
-    if (_transaction != nil)
-        [information setValue:_transaction forKey:@"sentry-transaction"];
-
-    if (_userId != nil)
-        [information setValue:_userId forKey:@"sentry-userid"];
-
     if (_userSegment != nil)
-        [information setValue:_userSegment forKey:@"sentry-usersegment"];
+        [information setValue:_userSegment forKey:@"sentry-user_segment"];
+
+    if (_sampleRate != nil)
+        [information setValue:_sampleRate forKey:@"sentry-sample_rate"];
```

Comment:
Are we making sure here that the `sentry-sample_rate` is set as a [simple decimal number](https://github.com/getsentry/publish/issues/1203) (no exponential notation, NaN [if that's a thing in obj-c], etc)? (this applies to both, baggage and envelope header)

Reply:
Kind yes, because OBJ-C converter does not use exponential notation and dont have the NaN concept. So I would say we are good here.

Reply:
Awesome!